doc/usersguide/source/changes/version_0_12.rst.inc

```diff
@@ -43,6 +43,21 @@
 * implemented experimental align(link=) to use proportions in combination with the Chenard algorithm. It needs more
   testing though.
 
+* implemented a way of declaring global constants. These constant should be explicitely typed. (partially closes :issue:`203`): ::
```

Comment:
why partially?

Reply:
We cannot use:
````
    INT_CONSTANT: 42
````

Reply:
I consider the feature implemented, even though the syntax is not ideal. We can still improve the syntax later, but I will close the issue anyway.